client/hosting-overview/components/active-domains-card.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Card } from '@automattic/components';
+import { useSiteDomainsQuery } from '@automattic/data-stores';
+import { DomainsTable } from '@automattic/domains-table';
+import { useTranslate } from 'i18n-calypso';
+import { FC } from 'react';
+import { fetchSiteDomains } from 'calypso/my-sites/domains/domain-management/domains-table-fetch-functions';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+const ActiveDomainsCard: FC = () => {
+	const site = useSelector( getSelectedSite );
+	const { data, isLoading } = useSiteDomainsQuery( site?.ID, {
+		queryFn: () => fetchSiteDomains( site?.ID ),
+	} );
+	const translate = useTranslate();
+
+	return (
+		<Card className="hosting-overview__card">
+			<div className="hosting-overview__card-header">
+				<h3 className="hosting-overview__card-title">{ translate( 'Active domains' ) }</h3>
+
+				<Button className="hosting-overview__link-button" plain href="/start/domain">
```

Comment:
Maybe this should link to `/domains/add/:site` since the user most likely will like to add a new domain to their site?

Reply:
We discussed it on the issue (https://github.com/Automattic/dotcom-forge/issues/6384#issuecomment-2047943713), so I'll leave it like this for the moment, but it can be easily changed at any point.